src/main/java/net/javadiscord/javabot/systems/qotw/submissions/subcommands/QOTWReviewSubcommand.java

```diff
@@ -81,12 +81,13 @@ public void execute(@NotNull SlashCommandInteractionEvent event) {
 		submission.retrieveAuthor(author -> {
 			SubmissionManager manager = new SubmissionManager(qotwConfig, pointsService, questionQueueRepository, notificationService, asyncPool);
 			if (state.contains("ACCEPT")) {
-				manager.acceptSubmission(event.getHook(), submissionThread, author, state.equals("ACCEPT_BEST"));
+				manager.acceptSubmission(submissionThread, author, state.equals("ACCEPT_BEST"));
 			} else {
 				// just do a "wrong answer" for now. this command is going to be removed
 				// in the near future anyway
-				manager.declineSubmission(event.getHook(), submissionThread, author, SubmissionStatus.DECLINE_WRONG_ANSWER);
+				manager.declineSubmission(submissionThread, author, SubmissionStatus.DECLINE_WRONG_ANSWER);
 			}
+			Responses.success(event.getHook(), "Submission Accepted", "Successfully accepted submission by " + author.getAsMention()).queue();
```

Comment:
This message is sent regardless of the submission actually being accepted or not